editor/project_settings_editor.cpp

```diff
@@ -298,6 +304,10 @@ void ProjectSettingsEditor::_add_feature_overrides() {
 		ee->get_export_platform(i)->get_platform_features(&p);
 		for (const String &E : p) {
 			presets.insert(E);
+			if (E == "linuxbsd") {
+				presets.insert("linux");
+				presets.insert("bsd");
```

Comment:
Why not just add `linux`, `bsd`, `freebsd` etc to the value returned by `ee->get_export_platform(i)->get_platform_features(&p);`? It would make more sense to put the Linux and BSD specific logic inside of the export platform.

Reply:
Something like this

```cpp
void EditorExportPlatformLinuxBSD::get_platform_features(List<String> *r_features) const {
	EditorExportPlatformPC::get_platform_features(r_features);
#ifdef __linux__
	r_features->push_back("linux");
#else
	r_features->push_back("bsd");
#if defined(__FreeBSD__)
	r_features->push_back("freebsd");
#elif defined(__NetBSD__)
	r_features->push_back("netbsd");
#elif defined(__OpenBSD__)
	r_features->push_back("openbsd");
#endif
#endif
}
```

Reply:
I did this initially, but then I realized that this was a mistake. This causes both the `linux` and `bsd` tags to be added at the same time on export, but this makes no sense. And you cannot know in advance (when exporting a preset) which of these "runtime tags" will be in a particular case. Export for Linux/BSd can be done even under Windows, so conditional compilation won't help here. Also, it will hurt [my plugin](https://github.com/dalexeev/gdscript-preprocessor). :)

Reply:
I think it's worth adding a comment here.